.github/workflows/sql-release.yml

```diff
@@ -17,21 +18,26 @@ jobs:
           - os: linux
             arch: amd64
             runner: ubuntu-latest
-          - os: windows
-            arch: amd64
-            runner: windows-latest
+#          - os: windows
+#            arch: amd64
+#            runner: windows-latest
     runs-on: ${{ matrix.runner }}
     steps:
       - uses: actions/checkout@v3
+      - name: Increment patch version
+        run: |
+          cd sql
+          mvn --batch-mode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn validate -Pbump-patch
       - uses: graalvm/setup-graalvm@v1
         with:
           version: "22.1.0"
           java-version: "17"
           components: "native-image"
-      - name: Install Protoc
-        uses: arduino/setup-protoc@v1
-        with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
+#      - name: Install Protoc
+#        uses: arduino/setup-protoc@v1
+#        with:
+#          repo-token: ${{ secrets.GITHUB_TOKEN }}
+#          version: '21.5'
```

Comment:
Do you know why this installation option doesn't work? It seems to be widely used. We may also need protoc on the golang side soon, so would be nice to understand

Reply:
https://github.com/arduino/setup-protoc/issues/33

Reply:
Are we using any cutting-edge features or could we just downgrade to the highest version it supports until this issue is resolved?

Reply:
@pjain1 , I wonder if need that specific version?

Reply:
I just started with the latest stable and it was supported on Java side. For the test I installed protoc [manually](https://github.com/rilldata/rill-developer/blob/main/.travis.yml#L78) using the installation instructions given [here](http://google.github.io/proto-lens/installing-protoc.html).